src/content/developers/docs/scaling/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Scaling
+description: An introduction to the different scaling options currently being developed by the Ethereum community.
+lang: en
+sidebar: true
+incomplete: true
+---
+
+## Scaling Overview {#scaling-overview}
+
+The scalability of a blockchain is measured in transactions per second. On the Ethereum blockchain network congestion leads to slower transactions and nonviable gas prices. Increasing the network capacity (transactions per second) is fundamental to the meaningful and mass adoption of Ethereum. Conceptually scaling can be categorised as either on-chain scaling or off-chain scaling.
```

Comment:
Scalability isn't strictly just TPS. It can relate to node size/work too. Sharding allows scaling because nodes working on shard A only need to validate shard A not the entire network. 

Our eth2 pages dig into this. We should be wary of how we define scalability.

Reply:
Cheers for the feedback, I'll dig into this more over the weekend.